.github/CONTRIBUTING.md

```diff
@@ -16,11 +16,16 @@ cd vercel
 corepack enable
 pnpm install
 pnpm build
-pnpm lint
-pnpm test-unit
 ```
 
-Make sure all the tests pass before making changes.
+To run the tests, we'll need a `VERCEL_TOKEN`. To get that token:
+
+- Option 1: Login to https://vercel.com, go to Settings, Tokens, then create a "Full Account" token to use for this purpose.
+- Option 2: If you've logged in with the `vercel` CLI before, you can find the token value in `~/Library/Application Support/com.vercel.cli/auth.json`.
```

Comment:
This will be in a different location for windows. Probably best to omit option 2 here since its not advised to share the same token you use for development and testing

Reply:
On Windows the path is `%APPDATA%\com.vercel.cli\Data\auth.json`. `%APPDATA%` typically resolves to `C:\Users\<username>\AppData\Roaming`.